backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/TypeImpl.java

```diff
@@ -690,6 +730,33 @@ public boolean eIsSet(int featureID) {
         return super.eIsSet(featureID);
     }
 
+    /**
+     * @generated NOT
+     */
+    @Override
+    public EList<Membership> visibleMemberships(EList<Namespace> excluded, boolean isRecursive, boolean includeAll) {
+        // Inherited members are visible members of a type
+        BasicEList<Type> excludedTypes = excluded.stream()
+                .filter(Type.class::isInstance)
+                .map(Type.class::cast)
+                .collect(toCollection(BasicEList<Type>::new));
+
+        EList<Membership> visibleMemberships = super.visibleMemberships(excluded, isRecursive, includeAll);
+        EList<Membership> inheritedMemberships = inheritedMemberships(excludedTypes);
+        List<Membership> data = Stream.concat(visibleMemberships.stream(),
```

Comment:
Please rename `data` to something like `visibleMemberships`, and may be rename already existing `visibleMemberships` to something else.

Reply:
Done.